test/dartdoc_test_base.dart

```diff
@@ -70,23 +72,37 @@ analyzer:
         packagePath, name, Uri.file('$packagePath/'));
   }
 
+  Future<PackageGraph> _bootPackageFromFiles(FileGenerator files) async {
```

Comment:
I'm a little confused by the name. `files` doesn't seem like something you'd invoke. `fileGenerator`?

Reply:
Yes.  I'm a little confused by why I have a function as a parameter here at all.   Deleted that, so now the name makes sense.